corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -655,15 +655,23 @@ def get_case_types_for_domain(domain):
 
     @staticmethod
     def soft_delete_cases(domain, case_ids, deletion_date=None, deletion_id=None):
+        from corehq.form_processor.change_publishers import publish_case_saved
+
         assert isinstance(case_ids, list)
-        deletion_date = deletion_date or datetime.utcnow()
+        utcnow = datetime.utcnow()
+        deletion_date = deletion_date or utcnow
         with get_cursor(CommCareCaseSQL) as cursor:
             cursor.execute(
-                'SELECT soft_delete_cases(%s, %s, %s, %s) as affected_count',
-                [domain, case_ids, deletion_date, deletion_id]
+                'SELECT soft_delete_cases(%s, %s, %s, %s, %s) as affected_count',
+                [domain, case_ids, utcnow, deletion_date, deletion_id]
             )
             results = fetchall_as_namedtuple(cursor)
-            return sum([result.affected_count for result in results])
+            affected_count = sum([result.affected_count for result in results])
+
+        for case in CaseAccessorSQL.get_cases(case_ids):
+            publish_case_saved(case)
```

Comment:
i think there still might be a slight discrepancy in the couch vs sql models if we remove this - the couch model uses a bulk save but those changes still end up in the changes feed.  if we don't publish the change here then the deleted case won't get to the pillows.  is that going to be a problem?